src/styles/blocks/header.scss

```diff
@@ -0,0 +1,129 @@
+.header {
+  background-size: cover;
+  background-position: right 13% center;
+  background-image: url(../images/photos/header-bg.png);
```

Comment:
Too large image file size. Better use JPG format to optimize loading.

Reply:
fixed